FirebaseInAppMessaging/Sources/Public/FirebaseInAppMessaging/FIRInAppMessaging.h

```diff
@@ -42,6 +42,8 @@ NS_ASSUME_NONNULL_BEGIN
  *   2. Set `FirebaseInAppMessagingAutomaticDataCollectionEnabled` to false in the app's plist file.
  *   3. Global Firebase data collection setting.
  **/
+
+NS_EXTENSION_UNAVAILABLE_IOS("Firebase In App Messaging is not supported for iOS extensions.")
```

Comment:
cc @eldhosembabu same question

Reply:
Presumably extension builds have never worked for FIAM.